e2e-tests/tfm/cypress/e2e/pages/amendments/amendmentsPage.js

```diff
@@ -37,16 +39,57 @@ const amendmentsPage = {
   amendmentFacilityEndDateDayInput: () => cy.get('[data-cy="amendment--facility-end-date-day"]'),
   amendmentFacilityEndDateMonthInput: () => cy.get('[data-cy="amendment--facility-end-date-month"]'),
   amendmentFacilityEndDateYearInput: () => cy.get('[data-cy="amendment--facility-end-date-year"]'),
+  amendmentCurrentFacilityEndDate: () => cy.get('[data-cy="amendment--current-facility-end-date"]'),
+  amendmentFacilityEndDateDetails: () => cy.get('[data-cy="amendment--facility-end-date-details"]'),
+
+  amendmentBankReviewDateDayInput: () => cy.get('[data-cy="amendment--bank-review-date-day"]'),
+  amendmentBankReviewDateMonthInput: () => cy.get('[data-cy="amendment--bank-review-date-month"]'),
+  amendmentBankReviewDateYearInput: () => cy.get('[data-cy="amendment--bank-review-date-year"]'),
+  amendmentCurrentBankReviewDate: () => cy.get('[data-cy="amendment--current-bank-review-date"]'),
+  amendmentBankReviewDateDetails: () => cy.get('[data-cy="amendment--bank-review-date-details"]'),
```

Comment:
Not a blocker as there is an existing structure in this file - could have simpler naming e.g:

```js
const amendmentsPage = {
  newCoverEndDateDay: () => '...',
  coverEndDateDayInput: () => '...',
  coverEndDateMonthInput: () => '...',
  coverEndDateYearInput: () => '...',
  facilityValueInput: () => '...',
  currentFacilityValue: () => '...',
  // etc
}
```

So then tests have shorter and more readable selectors 😎 :

```js
amendmentsPage.newCoverEndDateDay

// instead of:
amendmentsPage.amendmentNewCoverEndDateDay
```